### DISCUSSION TITLE
SPX Test Assignments

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#7461] [Show and tell]
# ![banner](https://user-images.githubusercontent.com/63351350/166924244-845e103d-1ef9-4c4d-b9c3-7f5fdebf8851.png)

> Note: If you have any feedbacks on the test assignments, please leave a comment [here](https://github.com/filecoin-project/lotus/discussions/7462).

1️⃣ 0️⃣ 1️⃣ 0️⃣  [2022-05-04](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-2689653)
1️⃣ 0️⃣ 0️⃣ 1️⃣  [2022-05-02](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-2689631)
1️⃣ 0️⃣ 0️⃣ 0️⃣  [2022-04-13](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-2689610)
0️⃣ 1️⃣ 1️⃣ 1️⃣  [2022-03-31](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-2689589)
0️⃣ 1️⃣ 1️⃣ 0️⃣  [2022-03-25](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-2439618)
0️⃣ 1️⃣ 0️⃣ 1️⃣  [2022-03-02](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-2280626)
0️⃣ 1️⃣ 0️⃣ 0️⃣   [2022-02-14](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-2173382)
0️⃣ 0️⃣ 1️⃣ 1️⃣  [2021-11-30](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-1725696)
0️⃣ 0️⃣ 1️⃣ 0️⃣  [2021-11-17](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-1662198)
0️⃣ 0️⃣ 0️⃣ 1️⃣ [2021-10-14](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-1474538)
0️⃣ 0️⃣ 0️⃣ 0️⃣  [2021-10-06 ](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-1439902)

**Comment by jennijuju** [comment]
# 2021-10-07 🍫 

| # |  Cohort(s) | Lotus Version/Branch| Assignment |
| ------------- | ------------- |  ------------- | ------------- | 
| 0 | All | [lotus v1.13.0-rc2](https://github.com/filecoin-project/lotus/releases/tag/v1.13.0-rc2) | [General Testing and Call Outs](https://github.com/filecoin-project/lotus/discussions/7463) |
| 1 | All  | [lotus v1.13.0-rc2](https://github.com/filecoin-project/lotus/releases/tag/v1.13.0-rc2)| [Deal Making (⭐ graph sync v0.10.1)](https://github.com/filecoin-project/lotus/discussions/7465) |
| 2 | All | [lotus v1.13.0-rc2](https://github.com/filecoin-project/lotus/releases/tag/v1.13.0-rc2) | [Proof v10 CUDA support and pipeline optimizations](https://github.com/filecoin-project/lotus/discussions/7443) |
| 3 | TBD | Coming Soon... | Sector Storage(worker) groups |

**Reply by jennijuju** [reply]
# 2021-10-13 🍫 

| # |  Cohort(s) | Lotus Version/Branch| Assignment |
| ------------- | ------------- |  ------------- | ------------- | 
| 0 | All | [lotus v1.13.0-rc2](https://github.com/filecoin-project/lotus/releases/tag/v1.13.0-rc2)| [Deal Making (⭐ graph sync v0.10.1) - Follow up testing 2021-10-14](https://github.com/filecoin-project/lotus/discussions/7465#discussioncomment-1474524) |

This is an optional assignment, however, it'd be highly appreciated if we can get a couple of Fellow to participate and share the result soon.

**Comment by jennijuju** [comment]
# 2021-11-17  

| # |  Cohort(s) | Lotus Version/Branch| Assignment |
| ------------- | ------------- |  ------------- | ------------- | 
| 0 | All | [show-you-the-magik](https://github.com/filecoin-project/lotus/tree/show-you-the-magik) | [Sector Storage(worker) group](https://github.com/filecoin-project/lotus/discussions/7642) |

>Note: **show-you-the-magik** is based of v1.13.1-rc2 + #7453 + #7572.

**Comment by jennijuju** [comment]
# 2021-11-30

| # |  Cohort(s) | Lotus Version/Branch| Assignment |
| ------------- | ------------- |  ------------- | ------------- | 
| 0 | All | [v1.13.2-rc1](https://github.com/filecoin-project/lotus/releases/tag/v1.13.2-rc1) | [General Testing and Call Outs](https://github.com/filecoin-project/lotus/discussions/7719) |
| 1 | All |[v1.13.2-rc4](https://github.com/filecoin-project/lotus/releases/tag/v1.13.2-rc4)  | [Retrieval Improvements (coming soon](https://github.com/filecoin-project/lotus/discussions/7874))|
| 2 | space force | [v1.13.2-rc1](https://github.com/filecoin-project/lotus/releases/tag/v1.13.2-rc1) | Scheduler Enhancement (coming soon)|
| 3 | outerspace | TBD | Indexer Provider |

**Comment by jennijuju** [comment]
# 2022-02-14

| # |  Cohort(s) | Lotus Version/Branch| Assignment |
| ------------- | ------------- |  ------------- | ------------- | 
| 0 | spx-outerspace | [`master-spx.idxprov.rc-1`](https://github.com/filecoin-project/lotus/releases/tag/master-spx.idxprov.rc-1) | [SPX Index Provider Testing](https://github.com/filecoin-project/lotus/discussions/8087) |

**Comment by Reiers** [comment]
# 2022-03-2

| # |  Cohort(s) | Lotus Version/Branch| Assignment |
| ------------- | ------------- |  ------------- | ------------- | 
| 0 | spx-outerspace | [`master-spx.idxprov.rc-5`](https://github.com/filecoin-project/lotus/releases/tag/master-spx.idxprov.rc-5) | [SPX Index Provider Testing](https://github.com/filecoin-project/lotus/discussions/8087) |

**Comment by jennijuju** [comment]
# 2022-03-25

| # |  Cohort(s) | Lotus Version/Branch| Assignment |
| ------------- | ------------- |  ------------- | ------------- | 
| 0 | spx-spaceforce | [v1.15.1-rc2](https://github.com/filecoin-project/lotus/releases/tag/v1.15.1-rc2) | [FVM 0.5 Testing](https://github.com/filecoin-project/lotus/discussions/8334) |

**Comment by rjan90** [comment]
# 2022-03-31

| #   |      Cohort(s)      |    Lotus Version/branch   |    Assignment    |
|----------|:-------------:|------:|------:|
|  0  | spx-seedvault |  [post-worker-rc3](https://github.com/filecoin-project/lotus/releases/tag/post-worker-rc3)  |  [PoSt Worker Testing](https://github.com/filecoin-project/lotus/discussions/8375)  |

**Comment by rjan90** [comment]
# 2022-04-13

| #   |      Cohort(s)      |    Lotus Version/branch   |    Assignment    |
|----------|:-------------:|------:|------:|
|  0  | spx-current-class |  [v1.15.2-rc1](https://github.com/filecoin-project/lotus/releases/tag/v1.15.2-rc1)  |  [SPX - v1.15.2-rc1 Testing](https://github.com/filecoin-project/lotus/discussions/8484)  |

**Comment by rjan90** [comment]
# 2022-05-02

| #   |      Cohort(s)      |    Lotus Version/branch   |    Assignment    |
|----------|:-------------:|------:|------:|
|  0  | spx-current-class |  [v1.15.2-rc3](https://github.com/filecoin-project/lotus/releases/tag/v1.15.2-rc3)  |  [SPX - v1.15.2-rc3 Testing](https://github.com/filecoin-project/lotus/discussions/8579)  |

**Comment by rjan90** [comment]
# 2022-05-04

| #   |      Cohort(s)      |    Lotus Version/branch   |    Assignment    |
|----------|:-------------:|------:|------:|
|  0  | spx-seedvault |  [group-sdr-rc2](https://github.com/filecoin-project/lotus/releases/tag/group-sdr-rc2)  |  [SPX - Group multicore SDR jobs](https://github.com/filecoin-project/lotus/discussions/8598)  |
